### DISCUSSION TITLE
[Feature] Hide person should be avaiable directly and with an Shortcut (like H), also the other Entires of the Menue

### DISCUSSION THREAD

**Discussion by kingvuke** [immich-app/immich#6081] [Feature Request]
### The feature

I'm currently going through 20k people, and the number of clicks can be massively reduced.

To save the amount of clicks, please add this Entries directly on the Page, without clicking on the 3 dots, and additonally add and Keyboard Shortcut like H for Hide, M for Merge, U for Unhide etc. 

![grafik](https://github.com/immich-app/immich/assets/104008018/e980ac68-65a6-4e5f-811c-ff8af9fbc602)

### Platform

- [x] Server
- [x] Web
- [ ] Mobile

**Comment by daFreeMan** [comment]
Going on 2 years, it sadly looks like this won't be implemented. :(

I don't feel quite so bad, though, I've only got ~600 IDd people (so far) to hide.

**Comment by enVolt** [comment]
https://github.com/immich-app/immich/discussions/31619

Will take a dig at this.
